### DISCUSSION TITLE
Welcome to orthotope Discussions!

### DISCUSSION THREAD

**Discussion by augustss** [augustss/orthotope#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bsdshell** [comment]
Should we have more examples how to use the fantastic library?

**Reply by augustss** [reply]
Absolutely.  Please add some and make a pull request.
